nmpolicy/internal/lexer/lexer.go

```diff
@@ -148,7 +150,7 @@ func (l *lexer) lexIdentity() (*Token, error) {
 
 		if l.isEOF() || l.isSpace() {
 			return token, nil
-		} else if l.isDot() || l.isEqual() || l.isVerticalTwoDots() || l.isPlus() {
+		} else if l.isDot() || l.isEqual() || l.isVerticalTwoDots() || l.isPlus() || l.isPipe() {
```

Comment:
Same should be added for lexNumber isn't it?

Reply:
As said, if it's not a number we have finish.